### PR TITLE
HOCS-5652 - json changes to update HMPO business areas

### PR DESCRIPTION
**Pull request by vinothon84**
HOCS-5652 : This PR contains json changes to load both POGR HMPO and GRO business areas. Currently DECS have common business areas which is same across both HMPO and GRO case types. As part of this change, we would have separate business areas for POGR/POGR2 HMPO and GRO case complaints.

### CODE REVIEW

File: src/main/resources/screens/POGR_BUS_CONTRIBUTIONS_ADD.json [hunk 1]
```diff
@@ -8,12 +8,35 @@
         "required"
       ],
       "props": {
-        "choices": "POGR_CONTRIB_BUS_AREA"
+        "choices": "POGR_CONTRIB_BUS_AREA",
```
Comment (LiamMacP) [inline src/main/resources/screens/POGR_BUS_CONTRIBUTIONS_ADD.json:11]: Would this be better using the conditionalChoices mechanic here than introducing a whole new field?

Comment (LiamMacP) [reply src/main/resources/screens/POGR_BUS_CONTRIBUTIONS_ADD.json:11]: Especially as this is the only thing changing?

Comment (vinothon84) [reply src/main/resources/screens/POGR_BUS_CONTRIBUTIONS_ADD.json:11]: Apart from this - name and label shall be included as well. Do we really need to using condition to render all 3 properties (choices, name and label) ?

Comment (LiamMacP) [reply src/main/resources/screens/POGR_BUS_CONTRIBUTIONS_ADD.json:11]: No you just need to change the choices that are shown to the user.

Comment (vinothon84) [reply src/main/resources/screens/POGR_BUS_CONTRIBUTIONS_ADD.json]: I have incorporated your comment and it makes code tidy as well 👍

File: src/main/resources/screens/POGR_BUS_CONTRIBUTIONS_ADD.json [hunk 2]
```diff
@@ -1,41 +1,27 @@
 {
-  "props": {},
-  "title": "Add Contribution Request",
```
Comment (LiamMacP) [inline src/main/resources/screens/POGR_BUS_CONTRIBUTIONS_ADD.json:3]: Why have these been removed?

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**